resource_monitor/src/resource_monitor.c

```diff
@@ -2221,6 +2282,11 @@ int main(int argc, char **argv) {
 		    {"snapshot-file",   required_argument, 0, LONG_OPT_SNAPSHOT_FILE},
 		    {"snapshot-events", required_argument, 0, LONG_OPT_SNAPSHOT_WATCH_CONF},
 
+		    {"catalog-task-name", required_argument, 0, LONG_OPT_CATALOG_RECORD_ID},
```

Comment:
This is going to cause confusion -- the same concept is called `task-name` in the command line, `RECORD_ID` in the code, and `uuid` in the JSON.  Let's call it UUID throughout.